tests/zfs-tests/tests/functional/rsend/Makefile.am

```diff
@@ -53,7 +55,8 @@ dist_pkgdata_SCRIPTS = \
 	send_hole_birth.ksh \
 	send_invalid.ksh \
 	send_mixed_raw.ksh \
-	send-wR_encrypted_zvol.ksh
+	send-wR_encrypted_zvol.ksh \
+	send_doall.ksh
```

Comment:
You'll need to add this new test case to the `rsend` group in the `tests/runfiles/common.run` around line 931.  Otherwise it won't be include in the test runs.